crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -66,10 +66,12 @@ def restore_chef
       def restore_crowbar
         Crowbar::Backup::Base.restore_files.each do |source, destination|
           # keep the permissions of the files that are already in place
+          srcpath = @data.join("crowbar", source)
+          src_string = srcpath.directory? ? "#{srcpath.to_s}/." : srcpath.to_s
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (https://github.com/bbatsov/ruby-style-guide#no-to-s)